test/src/test/java/org/corfudb/integration/ServerRestartIT.java

```diff
@@ -810,4 +811,77 @@ public void testCorfuTableMultiIndexReconstruction() throws Exception {
         runtime2.shutdown();
         runtime3.shutdown();
     }
+
+    /**
+     * This test verifies that a stream is rebuilt from a checkpoint, whenever two valid checkpoints exist, but the
```

Comment:
I think this comment should say that the stream is rebuilt from the latest checkpoint (marked latest based on the snapshot it covers) even though there is an older checkpoint that appears later in the stream.

Reply:
Done.